kernels/test/op_native_group_norm_test.cpp

```diff
@@ -20,110 +20,140 @@ using exec_aten::ScalarType;
+         1.0625});
+    optional<Tensor> weight =
+        tf.make({6}, {4.5625, -2.8750, -0.6875, 0.5625, -2.0625, -2.7500});
+    optional<Tensor> bias =
+        tf.make({6}, {-0.5000, -2.7500, 1.1875, 3.6875, 3.8125, 4.6875});
+    double eps = 1e-5;
+    Tensor out0 = tf.zeros({5, 6, 2, 2});
+    Tensor out1 = tf.zeros({5, 3});
+    Tensor out2 = tf.zeros({5, 3});
+    Tensor out0_expected = tf.make(
+        {5, 6, 2, 2},
+        {3.419882,  6.578348,  -3.573864, -4.701888, -4.509254, -2.234663,
+         -4.082768, 2.172355,  0.838826,  2.270225,  0.416747,  0.636962,
+         3.207030,  3.687500,  4.333131,  3.041869,  5.547079,  1.649148,
+         0.674665,  1.220376,  7.156189,  6.168714,  6.896327,  6.740410,
+         3.509863,  -3.022041, -2.441427, 5.542011,  -0.794903, -0.886369,
+         -7.014627, 1.217361,  1.120617,  1.463606,  0.091652,  1.491045,
+         3.293219,  4.640229,  3.091168,  3.248319,  4.895990,  1.114683,
+         3.092597,  1.114683,  3.262238,  5.434066,  7.450763,  9.312329,
+         5.570122,  0.101119,  -2.444796, -6.499403, -5.446074, -6.337338,
+         -0.454995, 0.436269,  2.228491,  0.871598,  1.838385,  0.786793,
+         4.362284,  3.737805,  4.390039,  3.057817,  5.814659,  6.202621,
+         6.258044,  2.932658,  3.366583,  -0.623879, 4.475045,  3.588276,
+         -0.082914, -4.936279, 6.438795,  -2.357929, 0.714463,  -5.402106,
+         0.236606,  -5.879963, 1.176247,  1.021916,  2.333727,  0.520341,
+         4.275447,  3.549392,  2.896994,  4.275447,  6.120910,  5.298480,
+         6.195676,  5.784461,  2.033296,  1.833920,  1.485010,  2.531738,
+         3.193988,  2.532378,  -5.406940, -8.053379, -6.467402, -5.425139,
+         -1.395059, -1.325575, 0.266062,  1.622680,  1.606336,  1.230405,
+         2.809896,  3.893110,  4.601880,  3.425055,  4.374411,  8.283354,
+         3.494898,  2.029045,  6.088204,  4.915522,  1.136877,  2.700454});
+    Tensor out1_expected = tf.make(
+        {5, 3},
+        {-1.89843750,
+         1.62500000,
+         -0.09375000,
+         -1.91406250,
+         -0.49218744,
+         -0.02343750,
+         -0.77343756,
+         0.08593753,
+         -1.55468738,
+         -2.73437500,
+         1.07031238,
+         0.35937503,
+         0.34374997,
+         -0.77343750,
+         0.10937499});
+    Tensor out2_expected = tf.make(
+        {5, 3},
+        {0.79116172,
+         0.42708409,
+         0.30238494,
+         0.50903118,
+         0.31929117,
+         0.45128885,
+         0.33067191,
+         0.39473253,
+         0.42994878,
+         0.53187561,
+         0.29930803,
+         0.29000264,
+         0.38669431,
+         0.38038814,
+         0.75809801});
+    op_native_group_norm_out(
+        input, weight, bias, 5, 6, 4, 3, eps, out0, out1, out2);
+    if (DTYPE == ScalarType::Half || DTYPE == ScalarType::BFloat16) {
+      EXPECT_TENSOR_CLOSE_WITH_TOL(
+          out0,
+          out0_expected,
+          2e-1,
```

Comment:
That's 20%. This doesn't make me feel comfortable.

Reply:
group_norm is one of the ops that automatic mixed precision will autocast to float32: https://intel.github.io/intel-extension-for-pytorch/xpu/1.10.200+gpu/tutorials/features/amp.html

I think the norm ops are just particularly prone to roundoff error, but I'm certainly not a numerical analysis person.

Reply:
Dug in a little further. Here's the PR that originally made PyTorch group_norm support Half: https://github.com/pytorch/pytorch/pull/100234/files#diff-7927db349f568afca2de9b94d74ea5c3b8cb468cb6a433d0cc1e61e65c515a36

It looks like the test is atol=rtol=5e-3. I think it's reasonable to argue that if we can't get the tolerances to be broadly similar then we have a correctness issue and thus don't actually support Half. I'll see what I can do; this one might have to wait for code sharing.

Reply:
test doesn't pass with `atol=rtol=5e-3`. Holding off on group_norm until we have code sharing.